ansible/roles/postldap/defaults/main.yml

```diff
@@ -318,18 +335,18 @@ postldap__postfix__dependent_lookup_tables:
     connection: '{{ postldap__postfix_ldap_connection }}'
     search_base: '{{ postldap__ldap_people_dn|join(",") }}'
```

Comment:
This search base should also be `postldap__ldap_base_dn`, so that service accounts (under the ou=Hosts tree, for example) can have mailboxes as well.